### ISSUE TITLE
Multi-series drilldown charts with null-values

### ISSUE THREAD

**Issue by meineckej** [highslide-software/highcharts.com#3692]
Example: http://jsfiddle.net/ubesLke0/
Change that triggered it: set a top-level value to null (already done in example above):
... {  name: 'Fruits2',  y: null,      drilldown: 'fruits2'  } ...
To reproduce: click on a bar and then back again, an exception is thrown:
  Uncaught TypeError: Cannot read property 'hide' of undefined drilldown.js:8 
Note: I tried as a quickfix to check if point.graphic is defined in ColumnSeries.prototype.animateDrillupTo, but I got other errors, e.g. when clicking on the bar next to the null-bar
  Highcharts-modules.js:98 Uncaught TypeError: Cannot read property 'getBBox' of undefined

**Comment by watou** [comment]
I am also encountering this with a two-series stacked column that drills down to a single series, and on drillUp produces the same "Cannot read property 'hide' of undefined".  On drillUp, it draws only the first of the stacked series.

**Comment by watou** [comment]
I was able to make the issue go away by using drilldown.src.js instead of drilldown.js and changing line 328 from:

	point.graphic.hide();					

to:

	if (point.graphic) {
		point.graphic.hide();					
	}

using the same "check then call" test used by the subsequent lines for dataLabel and connector.  A similar change could be made to line 8 of the minified version of the .js file.  Will this work for you, @meineckej ?

**Comment by watou** [comment]
It looks like my change is part of the change made for issue #3407 , but someone reports that issue #3407 is still a problem (but it deals with map drillups).

**Comment by meineckej** [comment]
Thanks, @watou.
In my hotfix, I also did what you desribed. but in addition, I also had to do this a few lines further down (aroudn line 340):
 
if (point.graphic) {
   point.graphic[verb](inherit);
}
                        
In addition, around line 122, I had to do this:
shapeArgs: point.shapeArgs,
            bBox: point.graphic ? point.graphic.getBBox() : null,

Drill-down and drill-up works for me with these changes, but when I drill down into a bar in the same group as the null value, I still get a legend with an unwanted "series 4" label for the null dimension member (that does not have series attached to it). Not sure if this is related to how we feed the drill-down series into the drilldown charts or not.
